FlowCrypt/Functionality/WKDURLs/PubLookup.swift

```diff
@@ -34,20 +31,17 @@ class PubLookup {
                 resolve([])
                 return
             }
+
             let wkdResult = try awaitPromise(self.wkd.lookupEmail(email))
             if !wkdResult.isEmpty {
                 resolve(wkdResult)
             }
+
             let attesterResult = try awaitPromise(self.attesterApi.lookupEmail(email: email))
-            guard let armoredData = attesterResult.armored,
-                  let attesterKeys = try? self.core.parseKeys(armoredOrBinary: armoredData),
-                  !attesterKeys.keyDetails.isEmpty else {
+            if attesterResult.isEmpty {
                 resolve([])
-                return
             }
```

Comment:
I suspect this part can be removed now, since it will default below to `resolve([])` anyway?

Reply:
oops, you are right